packages/client/components/NewMeetingTeamPickerAvatars.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React, {useMemo} from 'react'
+import {useFragment} from 'react-relay'
+import {PALETTE} from '../styles/paletteV3'
+import defaultUserAvatar from '../styles/theme/images/avatar-user.svg'
+import {NewMeetingTeamPickerAvatars_team$key} from '../__generated__/NewMeetingTeamPickerAvatars_team.graphql'
+import Avatar from './Avatar/Avatar'
+import ErrorBoundary from './ErrorBoundary'
+
+const Container = styled('div')<{count: number}>(({count}) => ({
+  width: 44,
+  display: 'flex',
+  justifyContent: count > 1 ? 'flex-start' : 'center',
+  flexWrap: 'wrap',
+  bottom: count > 2 ? 4 : 0,
+  position: 'relative'
+}))
+
+const AvatarWrapper = styled('div')<{count: number}>(({count}) => ({
+  width: count > 1 ? 20 : 'auto',
+  height: count > 2 ? 20 : 'auto'
+}))
+
+const StyledAvatar = styled(Avatar)({
+  border: `2px solid ${PALETTE.SLATE_200}`
+})
+
+const getShuffledArr = (arr) => {
```

Comment:
+1 maybe move `getShuffledArr.ts` to `/client/utils/` and use it on server and client?